src/main/java/org/mskcc/oncokb/curation/service/GeneService.java

```diff
@@ -200,8 +200,8 @@ public List<Gene> findByHugoSymbolIn(Set<String> searchValues) {
         return geneRepository.findByHugoSymbolInIgnoreCase(searchValues.stream().collect(Collectors.toList()));
     }
 
-    public Page<Gene> search(String query, Pageable pageable) {
-        Page<Gene> genePage = geneRepository.searchGene(query, pageable);
+    public Page<Gene> blurSearchByHugoSymbol(String query, Pageable pageable) {
```

Comment:
Where is this method use? Or do you plan on using it?

Reply:
Not used, just want to add an example on how to do blur search on repository level.